.prettierignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
No, it didn't work for me. Maybe because I opened it as part of a multi-root workspace?

I think it will also not work if kirbyup is in a subdirectory and `prettier` is run from the parent, or if prettier is not run via the VS Code extension at all, but via the CLI, the ESLint plugin or the integration of another editor like WebStorm/IntelliJ. `.prettierignore` should be a lot more robust.

(a key in `package.json` would be a lot cleaner than an extra file, but I'm waiting for https://github.com/prettier/prettier/issues/3460 since forever 😞)

Reply:
I see. With multi-root workspaces, the project-specific `.vscode` folder is ignored.